tests/cases/conformance/classes/members/privateNames/privateNameInInExpression.ts

```diff
@@ -0,0 +1,113 @@
+// @strict: true
+// @target: esnext
```

Comment:
Is this just for testing types and control flow? If not I'd recommend the following change so that we can see the same code transformed for each target:

```suggestion
// @target: esnext, es2015
```

Reply:
Yes this goal for this case file was to only test types and control flow. No transformation.